src/boost.beast/helper.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include <fmt/format.h>
+
+namespace beast = boost::beast;// from <boost/beast.hpp>
+namespace http = beast::http;// from <boost/beast/http.hpp>
+namespace net = boost::asio;// from <boost/asio.hpp>
```

Comment:
In general, I am not a big fan of redefining namespaces in headers. With `beast` and `http` I can live, but `net` for asio may be confusing as it's a completely different name.

Reply:
changed net to -> asio